src/test/java/com/bitmovin/api/examples/CreatePerTitleEncoding.java

```diff
@@ -0,0 +1,231 @@
+        {
+            System.out.println("Error in encoding. Exiting...");
+            return;
+        }
+
+
+        DashManifest dashManifest = new DashManifest();
+        dashManifest.setName("perTitleManifest.mpd");
+        dashManifest.setOutputs(Collections.singletonList(
+                new EncodingOutput(
+                        s3Output.getId(),
+                        OUTPUT_BASE_PATH
+                )
+        ));
+        dashManifest = bitmovinApi.manifest.dash.create(dashManifest);
+
+        Period period = new Period();
+        period = bitmovinApi.manifest.dash.createPeriod(dashManifest, period);
+
+        VideoAdaptationSet videoAdaptationSet = new VideoAdaptationSet();
+        videoAdaptationSet = bitmovinApi.manifest.dash.addVideoAdaptationSetToPeriod(dashManifest, period, videoAdaptationSet);
+
+        List<Muxing> perTitleMuxings = bitmovinApi.encoding.muxing.getMuxings(encoding);
+        for (Muxing muxing : perTitleMuxings)
+        {
+            if (CollectionUtils.isEmpty(muxing.getStreams()))
+            {
+                System.out.println(
+                        String.format("Found muxing with id %s that does not contain a stream...", muxing.getId())
+                );
+                continue;
+            }
+
+            Stream stream = bitmovinApi.encoding.stream.getStream(encoding, muxing.getStreams().get(0).getStreamId());
+            if (stream == null)
+            {
+                System.out.println(
+                        String.format("Could not get stream with id %s", muxing.getStreams().get(0).getStreamId())
+                );
+                continue;
+            }
+
+            if (StringUtils.isBlank(stream.getCodecConfigId()))
+            {
+                System.out.println(
+                        String.format("Could not find codec configuration id in stream with id %s", stream.getId())
+                );
+                continue;
+            }
+
+            H264VideoConfiguration h264VideoConfiguration = bitmovinApi.configuration.videoH264.get(stream.getCodecConfigId());
+            if (h264VideoConfiguration == null || h264VideoConfiguration.getBitrate() == null || h264VideoConfiguration.getBitrate() < 0)
+            {
+                System.out.println(
+                        String.format(
+                                "Could not get codec configuration of stream %s",
+                                stream.getId()
+                        )
+                );
+                continue;
+            }
+
+            DashFmp4Representation representation = new DashFmp4Representation();
+            representation.setMuxingId(muxing.getId());
+            representation.setStreamId(muxing.getStreams().get(0).getStreamId());
+            representation.setEncodingId(encoding.getId());
+            representation.setSegmentPath(
+                    String.format("video/%s/", h264VideoConfiguration.getBitrate())
```

Comment:
Don't we get the chosen output path back from the stream and muxing that get generated for per-title? This seems strange to do it like this.